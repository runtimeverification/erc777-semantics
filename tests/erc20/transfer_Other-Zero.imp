test.preamble();

// Testing transfer(To,Value) when To != caller and Value = 0.

test.transfer(8, 0);
