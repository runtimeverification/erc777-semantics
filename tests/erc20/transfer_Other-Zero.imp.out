7 is transferring 0 to 8
Balance of 7 is 70
Balance of 8 is 80
Allowance of 7 for 8 is 0
